### PR TITLE
[MM-45121] New team members widget

### PR DESCRIPTION
**Pull request by BenCookie95**
#### Summary
Adding UI for new members, this includes pagination for the modal

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-45121

#### Related Pull Requests

- [Has server changes (please link here)](https://github.com/mattermost/mattermost-server/pull/20708)

#### Screenshots
Screen Shot 2022-08-05 at 11 54 48 AM

Screen Shot 2022-08-05 at 11 55 54 AM

#### Release Note

```release-note
New members widget for insights
```

### CODE REVIEW

File: components/activity_and_insights/insights/top_dms_and_new_members/top_dms_table/top_dms_table.tsx
Comment (mkraft) [inline components/activity_and_insights/insights/top_dms_and_new_members/top_dms_table/top_dms_table.tsx:192]: Are these all meant to be hard-coded or no-ops?

File: reducers/plugins/index.ts
Comment (BenCookie95) [reply reducers/plugins/index.ts]: Not sure where you saw these changes? Not appearing in my files changed

File: components/widgets/users/avatar/avatar.tsx
Comment (BenCookie95) [reply components/widgets/users/avatar/avatar.tsx]: Not sure where you saw these changes? Not appearing in my files changed

File: components/root/root.jsx
Comment (BenCookie95) [reply components/root/root.jsx]: Not sure where you saw these changes? Not appearing in my files changed

File: components/activity_and_insights/insights/top_dms_and_new_members/top_dms_table/top_dms_table.tsx
Comment (BenCookie95) [reply components/activity_and_insights/insights/top_dms_and_new_members/top_dms_table/top_dms_table.tsx:192]: These are meant to be hardcoded because our pagination is quite different

**Review by mkraft — APPROVED** [review]

**Review by mkraft — APPROVED** [review]

**Review by mkraft — APPROVED** [review]

**Review by plant99 — APPROVED** [review]

**Review by matthewbirtch — APPROVED** [review]
Awesome work @BenCookie95. As discussed, the truncation and the modal hover state needs to be fixed and you mentioned you'll do that before merging. I'm going to approve to unblock this

**Comment by furqanmlk** [comment]
### 1. Changes in Profile does not reflect until page is refreshed
### 2. Profile picture is not showing, even after refresh. User might have to logout and login to see it

image


### 3. Position size should be 12px.  

image

### FIGMA
image


### 4. Profile pictures are not aligned when user **Position** is added 

image

### 5. Model window is going outside the border

image

### 6. Position column value should be truncated on Model window

image

### 7. Left arrow should be disabled on first page and right should be disabled on last page ?

image

image


### 8. Paging number should match with Figma

**FIGMA**
image


**ACTUAL**
image

**Review by furqanmlk — COMMENTED** [review]
.

**Comment by BenCookie95** [comment]
> ### 1. Changes in Profile does not reflect until page is refreshed
> ### 2. Profile picture is not showing, even after refresh. User might have to logout and login to see it
> image
> 
> ### 3. Position size should be 12px.
> image
> 
> ### FIGMA
> image
> 
> ### 4. Profile pictures are not aligned when user **Position** is added
> image
> 
> ### 5. Model window is going outside the border
> image
> 
> ### 6. Position column value should be truncated on Model window
> image
> 
> ### 7. Left arrow should be disabled on first page and right should be disabled on last page ?
> image
> 
> image
> 
> ### 8. Paging number should match with Figma
> **FIGMA** image
> 
> **ACTUAL** image
@furqanmlk 
1. Will have to create a ticket for that
2. This is a server issue, need the `last_picture_update` field in order to do this. Will have to create a bug ticket
3. Will fix
4. Will fix
5. This is a side effect of #6 but there is already a ticket for this, only occurs on very small screens
6. Will fix
7. They are already disabled, maybe the hover effect is confusing things
8. All the other modals are 1-10 so I'm going to keep it this way, not sure why the figma is 1-9

**Comment by furqanmlk** [comment]
> thanks @furqanmlk
> 
> 1. Will have to create a ticket for that
> 2. This is a server issue, need the `last_picture_update` field in order to do this. Will have to create a bug ticket
> 3. Will fix
> 4. Will fix
> 5. This is a side effect of # 6 but there is already a ticket for this, only occurs on very small screens
> 6. Will fix
> 7. They are already disabled, maybe the hover effect is confusing things
> 8. All the other modals are 1-10 so I'm going to keep it this way, not sure why the figma is 1-9

Thanks @BenCookie95 
I have logged the bugs for Issue 1 and 2

**Comment by BenCookie95** [comment]
> > thanks @furqanmlk
> > 
> > 1. Will have to create a ticket for that
> > 2. This is a server issue, need the `last_picture_update` field in order to do this. Will have to create a bug ticket
> > 3. Will fix
> > 4. Will fix
> > 5. This is a side effect of # 6 but there is already a ticket for this, only occurs on very small screens
> > 6. Will fix
> > 7. They are already disabled, maybe the hover effect is confusing things
> > 8. All the other modals are 1-10 so I'm going to keep it this way, not sure why the figma is 1-9
> 
> Thanks @BenCookie95 I have logged the bugs for Issue 1 and 2

@furqanmlk Fixed 3,4, 6 and 7. 1, 2 & 5 have tickets and # 8 isn't an issue

**Review by furqanmlk — APPROVED** [review]
Thanks @BenCookie95 

Working as expected

**Pull request merged**
